### DISCUSSION TITLE
How do I know if my emails are all being forwarded correctly?

### DISCUSSION THREAD

**Discussion by jis5ey** [anonaddy/anonaddy#235] [Q&A] [answered]
Hello, 

I'm new to AnonAddy.  I've been a Blur/Abine user for many years to manage my disposable email addresses (aka "masked emails" in Abine's terminology) and never had any issue before. They have a temporary inbox where I can see/read all emails before they get deleted after 24 hours. That way, if I'm in a hurry to get an email to be received in my main mailbox, I can read it first on the Abine web interface (because there's a small delay before it gets forwarded). I can also manually force an email to be forwarded again. Thanks to this internal mailbox, recently, I've realized that a lot of my emails are not being automatically forwarded anymore,. I mean I compare which emails are in my temporary Abine mailbox and which emails are well received in my main mailbox, but now I can see some emails are missing (of course, I waited long enough for mail delivery, I've checked the spam folder and searched everywhere in my main email address). 

Because of that, I'm looking for an alternative to Blur/Abine and yesterday I've discovered AnonAddy but I don't know if I should switch to it because I'm afraid I could miss some emails again and not have a way to read them (like I can with Blur currently). I can't find any inbox or records about my emails from AnonAddy (even some basic metadata like sender and subject would be useful to me), only a counter about how many emails in total have been forwarded. That's why I've created this new discussion with the following question: "How do I know if my emails are all being forwarded correctly?". Also, maybe if there's any service down time with AnonAddy, do we get informed somehow?

**Comment by willbrowningme** [comment, marked as answer]
If AnonAddy attempts to forward an email to you but it is rejected (bounced) then you will receive a failed delivery notification.

You can check the [status page](https://status.anonaddy.com/) to see if AnonAddy is operating as expected.

**Reply by jis5ey** [reply]
Thanks a lot! I didn't know about this page. It's excellent, bookmarked!
